include/arch/riscv/arch/model/statedata.h

```diff
@@ -35,8 +37,47 @@ NODE_STATE_END(archNodeState);
 
 extern asid_pool_t *riscvKSASIDTable[BIT(asidHighBits)];
 
+/* Kernel Page Tables */
 /* RISC-V has n-level page tables (depending on configured paging mode),
- * each of which is 4KiB, PTEs are of word_t size
-  */
-extern pte_t kernel_pageTables[CONFIG_PT_LEVELS][BIT(PT_INDEX_BITS)] ALIGN(BIT(seL4_PageTableBits));
+ *  As of version 1.10,
+ *  - level 1 are entries for 1GiB pages
+ *  - level 2 are entries for 2MiB pages
+ *  - level 3 are entries for 4KiB pages
```

Comment:
This comment doesn't apply to the intended "general" n-level design of the kernel page tables that can work for Sv32, Sv39, Sv48 or future ones. Maybe move to a PT == 3 ```#ifdef```?

Reply:
I'll move the comment into the CONFIG_PT_LEVELS == 3 section immediately below this.